Library/Homebrew/extend/os/ENV/std.rb

```diff
@@ -0,0 +1,5 @@
+require "ENV/std"
```

Comment:
And the test now disclosed, what I had also missed, namely that this should have been `extend/ENV/std`. Which naturally raises the question how logical our hierarchy is or whether this file should be named `extend/os/extend/ENV/std.rb` instead of `extend/os/ENV/std.rb`.